pkgs/applications/graphics/digikam/default.nix

```diff
@@ -109,7 +112,10 @@ mkDerivation rec {
     marble
     oxygen
     threadweaver
-  ];
+  ] ++ lib.optionals cudaSupport (with cudaPackages; [
+    cuda_nvcc
```

Comment:
Does this work without `cuda_nvcc`? Otherwise, does it work with `cuda_nvcc` in `nativeBuildInputs` instead?

Reply:
It does, and this is also the case for `mlt.  I've pushed a change for both to move `cuda_nvcc` to `nativeBuildInputs`.